Makefile

```diff
@@ -25,26 +25,37 @@ help: ## Display this help message.
 build.%: ## Builds the given docker service. Example: make build.datastore
 	docker compose build $*
 
-build: ## Builds the all docker services.
+build: ## Builds the all docker setup.
+	$(MAKE) install_node
 	docker compose build
 
 run-devserver: # Runs the dev-server
-	docker compose up dev-server -d --no-deps
-	$(MAKE) update.requirements
 	docker compose up angular-build -d
 	$(MAKE) update.package
+	docker cp oppia-angular-build:/app/oppia/node_modules .
+	docker compose stop angular-build
```

Comment:
it is a small optimisation in the docker setup, when we setup the docker setup for the first time, it fails and is unable to start because for the first time, we do not have any node modules in the local dir. so the angular-build container throws errors and the server is unable to start for the first time we setup the server. by just stopping the angular-build container just after copying the node modules to the local dir, and then starting all the services after that fixes that issue (also similar case for the CI jobs)

Reply:
this is a fix for an issue when we run the server for the very first time. the issue was for the fresh server, we don't have node_modules due to which angular-build throws errors. so I am starting the angular-build container, copying the node_modules to local, stopping the angular-build container, and after that starting the whole server which fixes the issue

Reply:
Add it to the tacking issue and resolve this thread once done!

Reply:
done